src/test/java/MathOperationsTests.java

```diff
@@ -0,0 +1,106 @@
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import javax.script.ScriptException;
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+public class MathOperationsTests {
+    private static final int FIRST_NUM_TASK_1 = 74;
+    private static final int SECOND_NUM_TASK_1 = 36;
+    private static final int EXPECTED_NUM_TASK_1 = 110;
```

Comment:
If some value is used only for one method - there is no need to extract it as a separate one

Reply:
INT_ARRAY_WITH_DUPLICATES and EXPECTED_DUPLICATES can be converted to local variables as well